Yatzy-Kata/Yatzy.Console/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Yatzy_Kata.Data;
+using Yatzy_Kata.Factories;
+using Yatzy_Kata.Interfaces;
+using Yatzy_Kata.Strategies;
+
+namespace Yatzy_Kata 
+{
+    public static class Program
+    {
+        static void Main(string[] args)
+        {
+            var player1 = new Player("bundy", new ComputerStrategy(new RandomDiceRoll()));
+            var player2 = new Player("james", new ComputerStrategy(new RandomDiceRoll()));
+            var player3 = new Player("bundy", new ConsoleUserStrategy(new ConsoleReader(), new ConsoleWriter(), new RandomDiceRoll()));
+
+            List<Player> players = new List<Player> { player1, player2, player3 };
+
+            var game = new Game(players, new RoundFactory(new TurnFactory()));
+            game.PlayGame();
```

Comment:
very clean and readable. A minor improvement could be to move the initialization (lines 13-15) into a separate service which pulls from a config file to make the game settings a bit more flexible

Reply:
A config model file and a service to provide the resources for the game is a great idea, and maybe even a controller that asks console to manage player count is a good idea. Will look into this further thank you.